configs/CelebA/fedavg_gan_dcgan.yml

```diff
@@ -0,0 +1,76 @@
+clients:
+    # Type
+    type: simple
+
+    # The total number of clients
+    total_clients: 2
+
+    # The number of clients selected in each round
+    per_round: 2
+
+    # Should the clients compute test accuracy locally?
+    do_test: false
+
+server:
+    address: 127.0.0.1
+    port: 8000
+
+    # Choose to send Generator or Discriminator to clients at
+    # the end of each round
+    network_to_sync:
+        generator: true
+        discriminator: false
```

Comment:
It may be a better idea to just say: `network_to_sync: generator` or `network_to_sync: discriminator`.

Reply:
Now updated in the latest commit